### PR TITLE
[zh] Rework securing cluster page

### PR DESCRIPTION
**Pull request by tengqm**
This page was translated poorly. Many texts were not properly translated.

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 1]
```diff
@@ -415,13 +420,15 @@ are not encrypted or an attacker gains read access to etcd.
 ### 对 Secret 进行静态加密
 
 一般情况下，etcd 数据库包含了通过 Kubernetes API 可以访问到的所有信息，
-并且可以授予攻击者对集群状态的可见性。
-始终使用经过良好审查的备份和加密解决方案来加密备份，并考虑在可能的情况下使用全磁盘加密。
+并且可能为攻击者提供对你的集群的状态的较多的可见性。
+你要始终使用经过良好审查的备份和加密方案来加密备份数据，
```
Comment (mengjiao-liu) [inline content/zh/docs/tasks/administer-cluster/securing-a-cluster.md:424]: “良好审查” 改成“充分审查” 或者“仔细审查” 会不会好一点？

Comment (tengqm) [reply content/zh/docs/tasks/administer-cluster/securing-a-cluster.md:424]: done

File: content/zh/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 2]
```diff
@@ -90,10 +94,10 @@ actions a client might want to perform. It is recommended that you use the [Node
 一旦通过身份认证，每个 API 的调用都将通过鉴权检查。
 Kubernetes 集成[基于角色的访问控制（RBAC）](/zh/docs/reference/access-authn-authz/rbac/)组件，
 将传入的用户或组与一组绑定到角色的权限匹配。
-这些权限将动作（get，create，delete）和资源（pod，service, node）在命名空间或者集群范围内结合起来，
-根据客户可能希望执行的操作，提供了一组提供合理的违约责任分离的外包角色。
-建议你将[节点](/zh/docs/reference/access-authn-authz/node/) 和
-[RBAC](/zh/docs/reference/access-authn-authz/rbac/) 一起作为授权者，再与
+这些权限将动作（get、create、delete）和资源（pod、service、node）进行组合，并可在名字空间或者集群范围生效。
```
Comment (howieyuen) [inline content/zh/docs/tasks/administer-cluster/securing-a-cluster.md:97]:
```suggestion
这些权限将动作（get、create、delete）和资源（Pod、Service、Node）进行组合，并可在名字空间或者集群范围生效。
```

File: content/zh/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 3]
```diff
@@ -170,14 +174,13 @@ reserved resources like memory, or to provide default limits when none are speci
 -->
 ### 限制集群上的资源使用
 
-[资源配额](/zh/docs/concepts/policy/resource-quotas/)
-限制了授予命名空间的资源的数量或容量。
-这通常用于限制命名空间可以分配的 CPU、内存或持久磁盘的数量，但也可以控制
-每个命名空间中有多少个 Pod、服务或卷的存在。
+[资源配额（Resource Quota）](/zh/docs/concepts/policy/resource-quotas/)限制了赋予命名空间的资源的数量或容量。
+资源配额通常用于限制名字空间可以分配的 CPU、内存或持久磁盘的数量，
+但也可以控制每个名字空间中存在多少个 Pod、服务或卷。
```
Comment (howieyuen) [inline content/zh/docs/tasks/administer-cluster/securing-a-cluster.md:179]:
```suggestion
但也可以控制每个名字空间中存在多少个 Pod、Service 或 Volume。
```

File: content/zh/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 4]
```diff
@@ -205,10 +208,10 @@ containers to run as a non-root user. Similarly, administrators who wish to prev
 client applications from escaping their containers should use a restrictive pod security
 policy.
 -->
-一般来说，大多数应用程序需要限制对主机资源的访问，
-他们可以在不能访问主机信息的情况下成功以根进程（UID 0）运行。
+一般来说，大多数应用程序需要对主机资源的有限制的访问，这样它们可以在不访问主机信息的情况下
```
Comment (howieyuen) [inline content/zh/docs/tasks/administer-cluster/securing-a-cluster.md:211]:
```suggestion
一般来说，大多数应用程序需要对主机资源的有限制的访问，
```

File: content/zh/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 5]
```diff
@@ -205,10 +208,10 @@ containers to run as a non-root user. Similarly, administrators who wish to prev
...
+成功地以 root 账号（UID 0）运行。
```
Comment (howieyuen) [inline content/zh/docs/tasks/administer-cluster/securing-a-cluster.md:212]:
```suggestion
这样它们可以在不访问主机信息的情况下，成功地以 root 账号（UID 0）运行。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Comment by tengqm** [comment]
@howieyuen All feedbacks taken. Please take another look.

**Pull request merged**